routes/weather.ts

```diff
@@ -338,7 +340,7 @@ export const getWateringData = async function( req: express.Request, res: expres
 			"&sunrise="		+	data.sunrise +
```

Comment:
I dont think `outputFormat === `json` is ever true. It doesnt look like Firmware ever sets this mode. We should confirm but I think we could remove L333-336:
```
	// Return the response to the client in the requested format
	if ( outputFormat === "json" ) {
		res.json( data );
	} else {
```

Reply:
This is used by the unit tests to make it easier to compare the results. Unless we improve the unit tests to read the strings parsed for the data needed I believe this may be better to keep around. I figure JSON output for the API should be the preferred solution over the URL query format so eventually other tools (such as the app for weather) should be using this type of output.